### ISSUE TITLE
cannot open include file google/protobuf/compiler/csharp/names.h when building grpc plugins

### ISSUE THREAD

**Issue by erictamlam** [grpc/grpc#38204]
### What version of gRPC and what language are you using?
v1.68.0 C++

### What operating system (Linux, Windows,...) and version?
Windows 10

### What runtime / compiler are you using (e.g. python version or version of gcc)
CMake GUI, MSVC 14.41.34120

### What did you do?
1) git clone https://github.com/grpc/grpc.git
2) cd grpc && git submodule update --init
3) cd third_party/protobuf && git checkout v3.12.3 // replace protobuf to v3.12.3 
4) CMAKE GUI Configure, uncheck all grpc plugins except for cpp and python 
5) CMAKE GUI Generate -> Open Project -> Build All

### What did you expect to see?
when building grpc_plugin_support, grpc_cpp_plugin, grpc_python_plugin, no building errors reporting missing protobuf header files

### What did you see instead?
cannot open include file: "google/protobuf/compiler/csharp/names.h"

Make sure you include information that can help us debug (full error message, exception listing, stack trace, logs).

See [TROUBLESHOOTING.md](https://github.com/grpc/grpc/blob/master/TROUBLESHOOTING.md) for how to diagnose problems better.

### Anything else we should know about your project / environment?
protoc_version: 3.12.3

**Comment by yijiem** [comment]
> 3. cd third_party/protobuf && git checkout v3.12.3 // replace protobuf to v3.12.3

I don't think you are supposed to change the protobuf version since protobuf introduced breaking changes between versions. Please use the same protobuf version that bundled with gRPC.
